src/user_management/user.model.js

```diff
@@ -23,17 +23,15 @@ export default (sequelize, DataTypes) => {
       type: DataTypes.STRING,
       allowNull: false
     }
-  },
-    {
-      instanceMethods: {
-        hashPassword = (password) => {
-          return bcrypt.hash(password, bcrypt.genSaltSync(10));
-        },
-        validatePassword = (password) => {
-          return bcrypt.compare(password, this.password);
-        }
-      }
-    })
+  })
+
+  User.prototype.validPassword = function (password) {
+    return bcrypt.compareSync(password, this.password)
+  }
+
+  User.prototype.hashPassword = function (password) {
+    return bcrypt.hash(password, bcrypt.genSaltSync(10))
+  }
 
   return User
 }
```

Comment:
You're missing something here. You need to hash the password each time it's changed and also on model instance creation. You can use Sequelize's Hooks for this purpose. You can read the documentation or ask Hiram about it.